.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,88 @@
+name: Issue report
+description: File an issue report
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: checkboxes
+  attributes:
+    label: Are you using any gamescope patches or a forked version of gamescope?
+    description: Please confirm any issues occur on upstream gamescope without any patches before filing an issue here.
+    options:
+    - label: The issue occurs on upstream gamescope without any modifications
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of the issue you're experiencing.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the issue.
+    placeholder: |
+      1. Launch Dota 2 from Steam with the gamescope launch command `gamescope -f -r 120 -- %command%`...
+      2. Enter a bot match
+      3. Move the cursor around
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Hardware information
+    description: |
+      examples:
+        - **Distro**: SteamOS 3.6.15 (`cat /etc/os-release`)
+        - **CPU**: 32-core AMD Ryzen Threadripper 7970X (`inxi` or `cat /proc/cpuinfo`)
+        - **GPU**: Advanced Micro Devices [AMD/ATI] Navi 31 [Radeon RX 7900 XT/7900 XTX/7900M] (`lspci -nn | grep VGA` or `lshw -C display -numeric` or `vulkaninfo --summary | grep deviceName`
```

Comment:
There's a right parenthesis missing here at the end of the line

Reply:
Will clean this up in my next housekeeping PR, thanks.